autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/UpdateFailedHandler.java

```diff
@@ -73,6 +81,8 @@ public void onApplicationEvent(UpdateFailedEvent event) {
             }
             suspendCluster(cluster);
             updateFailures.remove(autoscaleClusterId);
+            scalingActivityService.create(cluster, METRICS_COLLECTION_FAILED, messagesService.getMessageWithArgs(AUTOSCALE_YARN_RECOMMENDATION_FAILED,
```

Comment:
Will this be a new message? or should it be updating an existing one

Reply:
`METRICS_COLLECTION_FAILED` will be an initial message so this neds to be a new record